metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_config.py

```diff
@@ -80,10 +80,10 @@ class BigQueryV2Config(
     number_of_datasets_process_in_batch: int = Field(
         hidden_from_schema=True,
         default=500,
```

Comment:
In the description you said 200 without profiling, did you mean 500?

Reply:
ahh, yes, sorry